src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.smartdashboard.*;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+
+public class Shooter extends SubsystemBase {
+
+  private final TalonFX pivotMotor = new TalonFX(0); // TODO: Set to pivot motor device Id
+  private final TalonFX leftShootMotor = new TalonFX(0); // TODO: Set to left motor device Id
+  private final TalonFX rightShootMotor = new TalonFX(0); // TODO: Set to left motor device Id
+  private final TalonFX intakeMotor = new TalonFX(0); // TODO: Set to left motor device Id
```

Comment:
I need to use the intake motor to push the price from its holding state into the shooter, to fix that would the execute command just work with both subsystems?

Reply:
You got it! Though you can use both subsystems anywhere inside the subsystem except the constructor. This is because the constructor only adds the requirements for the subsystems, you can't do anything on it until at least after the scheduler calls `initialize`.